language/golang/index.md

```diff
@@ -29,5 +29,3 @@ You can containerize your own Go application using the examples and resources pr
 Let's get started!
 
 [Build your Go image](build-images.md){: .button .outline-btn}
-
-<br />
```

Comment:
I see these were all added at the end of the page; do we know if there was a reason for this; e.g. was (or _is_) there perhaps a styling issue to fix to have some whitespace at the end of our pages?

Reply:
I don't see why tbh:

Before:

![image](https://user-images.githubusercontent.com/1951866/179492503-924833de-4024-4888-b3d7-e238a066d49d.png)

After:

![image](https://user-images.githubusercontent.com/1951866/179492528-0653efd9-d45e-4aad-9254-585bd33c77b6.png)

Reply:
The only reason I can come up with would be something related to the left-hand or right-hand navigation, where it would try to scroll to the right location, and potentially ends up in a loop if it isn't able to scroll to that 🤔 

But perhaps it's not an issue